Bottom: 6e2dafa62c254449c1b64b4b7e5d00e6fb7c4539
Top:    0667e82c7c834831823a37d9003b019dbe75d946
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:41:09 +0000

Refresh of add-netbyname-to-the-select-ac

---

diff --git a/src/select.c b/src/select.c
index 007ff9d..eb5598b 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1017,6 +1017,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       InitConnectionLookup ();
       changed = ResetConnections (true, true) || changed;
 
+      SaveUndoSerialNumber ();
       MENU_LOOP (&PCB->NetlistLib);
       {
 	Cardinal i;
@@ -1033,6 +1034,8 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
 	  }
       }
       END_LOOP;
+      RestoreUndoSerialNumber ();
+
       changed = SelectConnection (Flag) || changed;
       changed = ResetConnections (true, true) || changed;
       FreeConnectionLookupMemory ();
